### PR TITLE
Add more detail to exceptions

### PR DESCRIPTION
**Pull request by msmerc**
When a changeUnit is missing an id / author it's helpful to know which!

(I've not tested this change!)

### CODE REVIEW

File: mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/internal/ChangeLogItem.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public ChangeLogItem(String id,
                        List<CHANGESET> beforeChangeSetsItems,
                        boolean system) {
     if (id == null || id.trim().isEmpty()) {
-      throw new MongockException("id cannot be null or empty.");
+      throw new MongockException("ChangeUnit["type.getName() + "] id cannot be null or empty.");
```
Comment (osantana85) [inline mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/internal/ChangeLogItem.java]: This line is not compiling, "+" is missing.

Comment (osantana85) [reply mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/internal/ChangeLogItem.java]: Also, for coherence with previous exception, could you change "ChangeUnit" to "changeUnit"?

**Review by osantana85 — CHANGES_REQUESTED** [review]
Hi @msmerc , thanks for contributing to Mongock. It's a good point to clarify which ChangeUnit is wrong. I've added a couple of comments as the code is not compiling. Please let us know when they are addressed and we are happy to merge it. Thanks.

File: mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/internal/ChangeLogItem.java (hunk 1 above)
Comment (msmerc) [reply mongock-core/mongock-runner/mongock-runner-core/src/main/java/io/mongock/runner/core/internal/ChangeLogItem.java]: Ok done.

Apologies for my laziness here - I'm trying to fix this code from the GitHub editor!

**Review by osantana85 — APPROVED** [review]
Thanks for your changes!

**Review by dieppa — APPROVED** [review]

**Pull request merged**